<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinMaxForm</class>
 <widget class="QMainWindow" name="MinMaxForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(125, 20, 60, 255), stop:1 rgba(0, 0, 0, 0));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <widget class="QLabel" name="label_matrix">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ваша матрица поиска максимина</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: transparent;</string>
    </property>
    <widget class="QPushButton" name="ButtonStrategy">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>440</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Выявить стратегию</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>501</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
       <hintingpreference>PreferDefaultHinting</hintingpreference>
      </font>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>75</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>75</number>
     </attribute>
    </widget>
   </widget>
   <widget class="QLineEdit" name="line_width">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: transparent;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(125, 20, 60, 255), stop:1 rgba(0, 0, 0, 0));
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_heigth">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: transparent;
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(125, 20, 60, 255), stop:1 rgba(0, 0, 0, 0));
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_width">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ширина матрицы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_heigth">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Высота матрицы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonTable">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Принять изменения</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
